src/api/routes/stockRoute.mjs

```diff
@@ -0,0 +1,11 @@
+import StockApiController from '../controllers/stockController.mjs';
+
+export default class StockRoute {
+
+    setStockRoutes(router) {
```

Comment:
Are all the routes going to be configured here for stocks?

Reply:
the routes will be set here, and the controllers will handle the logic associated to the routes